packages/cli/src/commands/generate/auth/providers/auth0.js

```diff
@@ -15,11 +14,15 @@ export const config = {
 }
 
 // required packages to install
-export const packages = ['@auth0/auth0-spa-js']
+export const webPackages = ['@auth0/auth0-spa-js']
+export const apiPackages = ['auth0']
```

Comment:
I don't think this package is required for our auth0 implementation?

Reply:
Ah, I see it's based on the get profile suggestion you've added - I think we should remove the automatic installation and suggest that the user can install it in your comment.